tests/seify.rs

```diff
@@ -152,13 +159,25 @@ fn config_cmd_map() -> Result<()> {
         let pmt = Pmt::MapStrPmt(HashMap::from([
             ("chan".to_owned(), Pmt::U32(0)),
             ("freq".to_owned(), Pmt::F64(102e6)),
-            ("gain".to_owned(), Pmt::F32(2.0)),
+            ("sample_rate".to_owned(), Pmt::F32(1e6)),
         ]));
-        fg_handle.callback(src, 3, pmt).await.unwrap();
+        fg_handle.callback(src, cmd_port_id, pmt).await.unwrap();
     });
 
     assert_approx_eq!(f64, dev.frequency(Rx, 0)?, 102e6, epsilon = 0.1);
-    assert_approx_eq!(f64, dev.gain(Rx, 0)?.unwrap(), 2.0);
+    assert_approx_eq!(f64, dev.sample_rate(Rx, 0)?, 1e6);
 
+    let conf = block_on(fg_handle.callback(src, cfg_port_id, Pmt::Ok))?;
+
+    match conf {
+        Pmt::VecPmt(v) => match v.as_slice() {
+            [Pmt::MapStrPmt(m), ..] => {
+                assert_eq!(m.get("freq").unwrap(), &Pmt::F64(102e6));
+                assert_eq!(m.get("sample_rate").unwrap(), &Pmt::F64(1e6));
+            }
+            o => panic!("unexpected pmt type {o:?}"),
+        },
+        o => panic!("unexpected pmt type {o:?}"),
+    }
```

Comment:
I coopted an existing test to test this functionality out. Can extract into a separate test if preferred.